modules/complete-onpremise-cluster/variables.tf

```diff
@@ -0,0 +1,16 @@
+variable "create" {
+    type = string
+    description = "create cluster"
```

Comment:
Why do we need to pass those values?
Terraform should be able to detect and provide right values to module to create/upgrade ...